dedupe/core.py

```diff
@@ -176,9 +177,29 @@ def scoreDuplicates(
     else:
         scored_pairs = numpy.array([], dtype=dtype)
 
+    # Monkeypatch in these extra methods and attributes.
```

Comment:
this is really cool!

Reply:
Yeah seriously! The first time I've used it too.